### DISCUSSION TITLE
Can network volumes be attached in task containers using mount?

### DISCUSSION THREAD

**Discussion by Anonymous-Coward** [concourse/concourse#5581] [Help & Support] [answered]
We need to generate a big file from a task step. It should ideally be written directly to the target network volume. Can a network drive be mounted from the task script? How?

The problem we are having is that we get a permission denied from mount no matter how we set the access rights to the network drive.

**Comment by cirocosta** [comment, marked as answer]
Hi @Anonymous-Coward ,

Have you tried with [`privileged: true`](https://concourse-ci.org/jobs.html#schema.step.task-step.privileged)? Also, could you provide an example that we could use to try reproducing?

thx!

**Reply by Anonymous-Coward** [reply]
privileged: true seems to have helped. We'll come back with more info if this proves to be insufficient.
